config/boards/licheepi-4a.wip

```diff
@@ -2,14 +2,15 @@
 BOARD_NAME="LicheePi 4A"
 BOARDFAMILY="thead"
 BOARD_MAINTAINER="chainsx"
-KERNEL_TARGET="legacy,edge"
+KERNEL_TARGET="legacy,current,edge"
 BOOT_FDT_FILE="thead/th1520-lichee-pi-4a.dtb"
 SRC_EXTLINUX="yes"
 SRC_CMDLINE="console=ttyS0,115200 rootwait rw earlycon clk_ignore_unused loglevel=7 eth=\$ethaddr tsched=0 rootrwoptions=rw,noatime rootrwreset=yes"
 BOOTCONFIG="light_lpi4a_defconfig"
 SKIP_BOOTSPLASH="yes"
 BOOTFS_TYPE="ext4"
 BOOTSIZE="512"
+EXTRAWIFI="no"
```

Comment:
Is there any particular reason why this is disabled? We usually disable them on exotic kernels, while mainline derived should just work. We keep them in shape.

Reply:
Thank you for your advice.